v0/template.json

```diff
@@ -61805,9 +61805,11 @@
             "type" : "object",
             "properties" : {
               "metadata" : {
-                "readOnly" : true,
```

Comment:
what is the implication of this readonly flag ?

Reply:
as for our use case, We need to update metadata field of already created ServicesYaml. So we had to remove @Value Annotation, That is why, schema generator is removing "readOnly" value from schema.